App.js

```diff
@@ -1,26 +1,111 @@
-import React from "react"
-import styled from "styled-components/native"
+import React, { useState } from 'react'
+import { TouchableOpacity, ScrollView } from 'react-native'
+import styled from "styled-components"
+
+import Icon from 'react-native-vector-icons/Feather'
+
+import TodoList from './Components/TodoList'
+
+
+
+export default function App() {
+  const [value, setValue] = useState('')
+  const [todos, setTodos] = useState([])
+
+  addTodo = () => {
```

Comment:
make sure to add the keyword `const` before creating the functions.

`const addTodo = () => { ... }`